src/webgpu/web_platform/util.ts

```diff
@@ -650,24 +650,38 @@ export const kObjectTypeFromFiles = [
  * convert the result to valid source that GPUCopyExternalImageSource supported.
  */
 export async function GetSourceFromImageFile(
+  test: GPUTest,
   imageName: ImageName,
   objectTypeFromFile: ObjectTypeFromFile
 ): Promise<ImageBitmap | HTMLImageElement> {
   const imageUrl = getResourcePath(imageName);
 
   switch (objectTypeFromFile) {
     case 'ImageBitmap-from-Blob': {
+      // MAINTENANCE_TODO: resource folder path when using service worker is not correct. Return
```

Comment:
Since this TODO affects coverage, please make sure it also has a corresponding non-MAINTENANCE `TODO` in a test file description or test description.